database/mssql/scripts/versions/v_7_ddl.sql

```diff
@@ -0,0 +1,66 @@
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+CREATE TABLE [permit].[ORBC_TRANSACTION](
+	[TRANSACTION_ID] [bigint] IDENTITY(1,1) NOT NULL,
+	[MERCHANT_ID] [int] NOT NULL,
+	[TRANSACTION_TYPE] [varchar] (3) NOT NULL,
+	[TRANSACTION_ORDER_NUMBER] [varchar](30) NOT NULL,
+	[TRANSACTION_AMOUNT] [decimal] (9, 2) NOT NULL,
+	[TRANSACTION_APPROVED] [tinyint] NOT NULL CHECK (TRANSACTION_APPROVED BETWEEN 0 AND 1),
+	[AUTH_CODE] [varchar] (32) NOT NULL,
+	[TRANSACTION_CARD_TYPE] [nvarchar](2) NOT NULL,
+	[TRANSACTION_DATE] [datetime2](7) NULL,
```

Comment:
I think we should maybe have another datetime column, indicating the datetime that the transaction was sent off to the payment provider. This will help us identify when the request was sent, for those requests that go out to Bambora and never come back (user closed browser, power outage, etc.). Could also help us when we're trying to avoid the user making duplicate payment attempts.

Reply:
I agree, and I will add an additional date for the send time.